x-pack/plugins/elastic_assistant/server/routes/route_validation.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { RouteValidationFunction, RouteValidationResultFactory } from '@kbn/core/server';
+import type { TypeOf, ZodType } from 'zod';
+import { stringifyZodError } from '@kbn/zod-helpers';
+
+export const buildRouteValidationWithZod =
```

Comment:
FYI, there's one of these over in the `kbn-elastic-assistant-common` package as well:

https://github.com/elastic/kibana/blob/c4545f5ce48e73c0449f4ac4fbabe18060a3b5c6/x-pack/packages/kbn-elastic-assistant-common/impl/schemas/common.ts#L42-L51

Reply:
resolved by https://github.com/elastic/kibana/pull/173487/commits/4cb61ecd5862957a0e4ac1454241a6e5228387c4